app/controllers/participations_controller.rb

```diff
@@ -16,6 +16,24 @@ def create
     end
   end
 
+  def edit
+    @outcome = Outcome.find(params[:outcome_id])
+    @participation = Participation.find(params[:id])
+    @available_years = ['2014', '2015', '2016', '2017', '2018', '2019']
```

Comment:
Can you think of a good way to eliminate this duplication? Could it be a view helper instead of an ivar? Maybe even calculate it in the helper so it doesn't have to be continually updated?